README.md

```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+# NC News - Front End
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This repo contains the front end development of the NC News app, using data obtained through my [NC News server](https://github.com/alexcupit/nc-news)
 
-## Available Scripts
+The site has been hosted and is live: https://acnews.netlify.app/
 
-In the project directory, you can run:
+## Description
 
-### `npm start`
+This site dispalys data from the northcoders news dataset.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+As a user, you can find various news articles, filtered by topic and ordered by date and their number of votes. Users can also vote on each article.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+Authenticated user creation is not needed for this demo site and to access authenticated functionality, you can choose from the list of already existing users on the log in screen.
```

Comment:
You currently have authorisation in place, but not authentication. Just a very minor point.